openshift/inspectBuildConfig-template.yaml

```diff
@@ -37,8 +37,8 @@ objects:
           memory: '512Mi'
           cpu: '500m'
         requests:
-          memory: '512Mi'
-          cpu: '500m'
+          memory: ${AMUN_BUILD_CPU}
+          cpu: ${AMUN_BUILD_MEMORY}
```

Comment:
Set both requests and limits to these values so we do not do possible over-commitment.